API/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require("supertest");
+const { seed } = require("../db/seeds/seed");
```

Comment:
Unless your seed file is exported in an object, you won't need to import it in an object